src/chatd.cpp

```diff
@@ -4743,6 +5001,9 @@ void Chat::setOnlineState(ChatState state)
 
     if (state == kChatStateOnline)
     {
+        // Send REACTIONSN after a reconnection
+        sendReactionSn();
```

Comment:
It should be sent even before the `HIST`/`JOINRANGEHIST`/`HANDLEJOIN`...
I'll fix it